teamcity: failed tests on master:

The following tests appear to have failed:

[#226197](https://teamcity.cockroachdb.com/viewLog.html?buildId=226197):

```

```
Please assign, take a look and update the issue accordingly.

Not sure what the root cause here is.

```
[18:25:12][Step 2/3] go test -v -installsuffix release -tags '' -ldflags ' -s -w -extldflags "-static-libgcc -static-libstdc++" -X github.com/cockroachdb/cockroach/pkg/build.typ=release' -i ./pkg/...
[18:25:13][Step 2/3] go generate -installsuffix release -tags '' -ldflags ' -s -w -extldflags "-static-libgcc -static-libstdc++" -X github.com/cockroachdb/cockroach/pkg/build.typ=release' ./pkg/...
[18:25:13][Step 2/3] stringer: checking package: cert.go:59:3: CreateCAPair not declared by package security
[18:25:13][Step 2/3] pkg/cli/context.go:170: running "stringer": exit status 1
```

Let's see if it comes back.

--- a/build/teamcity-check.sh
+++ b/build/teamcity-check.sh
@@ -5,9 +5,9 @@ export BUILDER_HIDE_GOPATH_SRC=1
 
 mkdir -p artifacts
 
-build/builder.sh make TYPE=release check 2>&1 | tee artifacts/check.log | go-test-teamcity
+build/builder.sh make check 2>&1 | tee artifacts/check.log | go-test-teamcity
 
-build/builder.sh make TYPE=release generate
+build/builder.sh make generate
 build/builder.sh /bin/bash -c '! git status --porcelain | read || (git status; git diff -a 1>&2; exit 1)'
 
 # Run the UI tests. This logically belongs in teamcity-test.sh, but we do it

